win32ss/drivers/videoprt/int10.c

```diff
@@ -266,7 +320,12 @@ IntInt10ReadMemory(
     INFO_(VIDEOPRT, "- Length: %x\n", Length);
 
     IntAttachToCSRSS(&CallingProcess, &ApcState);
+
```

Comment:
```suggestion
```

Reply:
Actually, I prefer to isolate the "attach-to-process" and the "handle-memory-copy" part.